src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/Panel.tsx

```diff
@@ -163,16 +153,12 @@ function Panel({
 
   return (
     <Widget location={breakpoint!.location} editor={editor} insertAt={insertAt}>
-      <div
-        onMouseEnter={onMouseEnter}
-        onMouseLeave={onMouseLeave}
-        style={{ position: "sticky", left: gutterOffset, maxWidth: width }}
```

Comment:
Oh that's interesting, we can just remove the "stick" style without it changing anything?

Reply:
Yeah, this will break the current functionality for sticking to the left side if the editor scroll area overflows.